### ISSUE TITLE
Design

### ISSUE THREAD

**Issue by sumeetjain** [omahacodeschool/goal-tracker#7]
Let's start talking about the design. @LeKohrs, what's your plan? Got any colors, fonts, logo ideas? This issue can be a conversation.

**Comment by LeKohrs** [comment]
Yep. All of those actually. I was going to show the group tomorrow. I can send you a pdf if you would like to see it before that.

**Comment by sumeetjain** [comment]
If you convert to a PNG/GIF/JPG, you can just upload it into a comment on this Issue. That's the best, since the group can see it and it's part of the conversation.

**Comment by LeKohrs** [comment]
![gettin -it-done](https://cloud.githubusercontent.com/assets/7944807/4560222/5b6db5ea-4ef3-11e4-85da-0826c7c9af2f.png)

Here is the sample. Basically this is the dashboard view. Each goal will be displayed on a "card" that will be placed either in rows of three of a side scroller view. Each card will have the title, description, the graph, a ticker for easy adding and a link to the show page for that goal. I can make a more traditional view as well if you are not interested in this. 

The Logo is in the top left. Thats up for debate, of course, but I thought it was fun.

Im open to whatever. These are just some initial ideas. Colors are samples as well, not set. Font is Helvetica lite.

**Comment by sumeetjain** [comment]
I like the colors quite a bit. And this design approach could be a good Phase 2, but for now I think it'd be best if you prepared a more traditional approach.

Let's make the logo a low priority, because the design is perfectly adequate without a logo at all. Let's go for minimalist, functional, and attractive.

**Comment by LeKohrs** [comment]
Sounds good.

**Comment by LeKohrs** [comment]
![gettin it done](https://cloud.githubusercontent.com/assets/7944807/4562434/57025ad6-4f04-11e4-8237-6d180f6ab11c.png)

Is this more what you were thinking of?
